app/assets/locales/android_translatable_strings.txt

```diff
@@ -857,4 +857,17 @@ quarantine.reason.server.processing=The server failed to process the form
 quarantine.reason.record.error=There was an issue with the form record that prevented submission
 quarantine.reason.manual=The form was manually quarantined by a user
 quarantine.reason.fnf=The file backing the form submission could not be found
-quarantine.reason.unknown=Reason for quarantine unknown
\ No newline at end of file
+quarantine.reason.unknown=Reason for quarantine unknown
+
+in.app.update.available.title=Update CommCare?

```

Comment:
the word "app" can be mistaken for the CommCare app that the user has installed on top of CommCare and therefore saying "app update" can confuse users in terms of what kind of update it is. We should strictly use word "CommCare" instead of saying "app" in all the user facing translations here.